api-gateway/api-composition/build.gradle.kts

```diff
@@ -43,5 +43,5 @@ tasks.named("build") {
     dependsOn("runTest")
 }
 
-apply(from = "https://raw.githubusercontent.com/Knotx/knotx-starter-kit/master/gradle/docker.gradle.kts")
-apply(from = "https://raw.githubusercontent.com/Knotx/knotx-starter-kit/master/gradle/javaAndUnitTests.gradle.kts")
\ No newline at end of file
+apply(from = "https://raw.githubusercontent.com/Knotx/knotx-starter-kit/2.1.0/gradle/docker.gradle.kts")
```

Comment:
Maybe we could add `knotxVersion` to settings and use `${project.property("knotxVersion")}` here?
So that future upgrade will have it in place.

Reply:
This change will be made when api-gateway projects are updated to version 2.2.0.

They are using api version 2.0.0 but 2.0.0 build script from starter-kit no longer works. I updated the script reference to fix build issue.